src/Surfnet/StepupSelfService/SelfServiceBundle/RemoteVetting/Configuration/RemoteVettingConfiguration.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
Why is RemoteVetting in a separate folder, should the Locale logic then also be moved to a Locale folder? Would you elaborate that the reason was to do so? Because I doubt if someone is making changes to the codebase in a few years from now if the separation is so clear?

Reply:
I see this as a collection of cohesive application parts (all related to RemoteVetting) collected in a namespace of their own. I find it convenient to have those classes together.